spec/features/moderation/proposals_spec.rb

```diff
@@ -2,6 +2,14 @@
 
 feature 'Moderate proposals' do
 
+  scenario 'Disabled with a feature flag' do
+    Setting['feature.proposals'] = nil
+    moderator = create(:moderator)
+    login_as(moderator.user)
+
+    expect{ visit moderation_proposals_path }.to raise_exception(FeatureFlags::FeatureDisabled)
```

Comment:
Same resetting of the feature flag value to true on this one as well (see previous comment)